test/run-end-to-end.py

```diff
@@ -31,7 +31,7 @@ def main():
     print("Providing Akri Helm chart with CRI args: {}".format(cri_args))
     extra_helm_args = shared_test_code.get_extra_helm_args()
     print("Providing Akri Helm chart with extra helm args: {}".format(extra_helm_args))
-    helm_install_command = "helm install akri {} --set debugEcho.enabled=true --set debugEcho.name={} --set agent.allowDebugEcho=true {} {} --debug ".format(helm_chart_location, shared_test_code.DEBUG_ECHO_NAME, cri_args, extra_helm_args)
+    helm_install_command = "helm install akri {} --set agent.full=true --set debugEcho.enabled=true --set debugEcho.name={} --set agent.allowDebugEcho=true {} {} --debug ".format(helm_chart_location, shared_test_code.DEBUG_ECHO_NAME, cri_args, extra_helm_args)
```

Comment:
does the build command for agent need to change for the e2e tests?

Reply:
(in .github/workflows/run-test-cases.yml ... do we need to change the command to build agent full?)

Reply:
Yes, made it so just agent-full builds and set the image repository to be the expected name `ghcr.io...agent-full`, so it should be what is pulled when `agent.full=true`. Thanks for pointing this out. Fixed in eb5b2f1464d01